src/client.cpp

```diff
@@ -736,7 +736,7 @@ Status Client::Impl::uploadObjectStream(id_t accountId, id_t instrumentId,
         return makeNetworkError();
     }
 
-    if (cs->getResponseCode() != 201)
+    if (cs->getResponseCode() != 201 && cs->getResponseCode() != 200)
```

Comment:
We can't say it as we do different operations: create and update with one HTTP command POST. It is broken from the very start. We can just make it work for now, by providing "hotfix"